_pages/cfp.md

```diff
@@ -5,7 +5,8 @@ subtitle: Call for Proposals
 permalink: /cfp/
 ---
 
-Our CFP will open on August 14 2017 and close on September 15 2017! 
+Our call for proposals (CFP) for 15 minute lightning talks will open on August 14 2017 and close on September 15 2017!
```

Comment:
My intention wasn't to make this a brainstorming thing. This was more of an FYI thing.